### PR TITLE
Sk rename ez crypto gem

### PR DESCRIPTION
**Pull request by skapfer**

### CODE REVIEW

**Comment by skapfer** [comment]
[Fix OpenSSL::Cipher::Cipher warnings](https://trello.com/c/idBazEd9/914-fix-opensslciphercipher-warnings)

**Comment by skapfer** [comment]
[Fix OpenSSL::Cipher::Cipher warnings](https://trello.com/c/idBazEd9/914-fix-opensslciphercipher-warnings)

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+require 'af_gems/appraisal'
+
+Rake::TestTask.new(:test) do |test|
+  test.libs << 'test'
+  test.pattern = 'test/**/*_test.rb'
+  test.verbose = true
+end
+
+namespace :test do
+  AfGems::RubyAppraisalTask.new(:all, [ 'ruby-2.3.3', 'ruby-2.5.3'])
```
Comment (spjsschl) [inline Rakefile]: it would be nice if that could appraise 2.5.3 and 2.6.3, you might need to change the Appraisal file as well

File: Appraisals [hunk 2]
```diff
@@ -1,5 +1,5 @@
 case(RUBY_VERSION)
-  when '2.3.3', '2.5.3' then
+  when '2.3.3', '2.5.3', '2.6.3' then
```
Comment (spjsschl) [inline Appraisals]: you can drop 2.3.3 here and in the Rakefile

File: ezcrypto.gemspec [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'ezcrypto/version'
+
 Gem::Specification.new do |s|
   s.name        = 'ezcrypto'
-  s.version     = '0.7.2'
-  s.date        = '2009-03-10'
+  s.version     = EzCrypto::VERSION
+  s.date        = '2019-05-31'
```
Comment (spjsschl) [inline ezcrypto.gemspec]: is this usually hardcoded in our other gems?

**Review by spjsschl — COMMENTED** [review]
looks pretty good. Couple of things:
1) make sure to add a .ruby-version
2) you do no longer need to appreiase ruby 2.3.3; you can remove it from Rakefile and Appraisals
3) we usually have a gemfiles directory which stores the gemfiles generated by appraisal for the different versions under test
4) you should add a cii build

File: ezcrypto.gemspec (hunk 3 above)
Comment (skapfer) [reply ezcrypto.gemspec]: I could not get gem bump to work without a version.rb file.
af_crypto, af_encrypted_attr, af_email all have version.rb.

Comment (spjsschl) [reply ezcrypto.gemspec]: yep. That is a requirement for AfGem pretty much. I was talking about the date though :-)

File: Appraisals (hunk 2 above)
Comment (skapfer) [reply Appraisals]: Done

File: Rakefile (hunk 1 above)
Comment (skapfer) [reply Rakefile]: Done

File: ezcrypto.gemspec (hunk 3 above)
Comment (skapfer) [reply ezcrypto.gemspec]: Oh. I removed the date, it's pointless

**Review by spjsschl — APPROVED** [review]

**Pull request merged**